Gemfile

```
@@ -5,10 +5,10 @@ end
 
 gemspec
 
-gem 'stackprof', platforms: :mri
-
 group :benchmark, :test do
   gem 'benchmark-ips'
+  gem 'memory_profiler'
+  gem 'stackprof', platforms: :mri
```

Since you're changing this line, may I suggest using `:install_if`..?
https://bundler.io/v1.10/man/gemfile.5.html#INSTALL_IF-install_if-

**For example,**
```suggestion
install_if -> { RUBY_PLATFORM !~ /mingw|mswin/ } do
  gem 'stackprof'
end
```